Updated primary key names

sbhuttan opened the pull request:
Updated primary keys names, as they were conflicting for two tables. You cannot create two keys with same name.

azurecla commented:
Hi __@padmavc__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

PRmerger commented:
@padmavc : Thanks for your contribution to the Azure documentation! The author, @msftman, has been notified to review your proposed change.

Erikre commented:
#sign-off

PRmerger commented:
Only the author of the article can use the sign off comment to add the ready-to-merge label.

padmavc commented:
#sign-off

sbhuttan committed:
Updated Primary Keys names

Updated primary keys names, as they were conflicting for two tables. You cannot create two keys with same name.
